File: python/game.py
import random

print("Welcome to the number guessing game imlemented in Python.")
print("Please guess a number that has been randomly chosen between 1 and 100.")
print("You only have 9 guesses, so make them count!")
print("Type 'quit' as your guess at any time to quit the game.")

random.seed()
random_num = random.randint(1, 100)

guess_count = 1

while guess_count < 10:
    guess_prompt = "\nGuess number {}: "
    guess_str = input(guess_prompt.format(guess_count))

    if guess_str == "quit":
        print("Bye!")
        guess_count = 11
    elif guess_str.isnumeric():
        guess_int = int(guess_str)

        if guess_int < 1 or 100 < guess_int:
            print("Please guess a number between 1 and 100")
        elif guess_int == random_num:
            print("Congradulations! You guesses the correct number!")
            guess_count = 11
        elif guess_int > random_num:
            print("Your guess is too high, try again!")
            guess_count += 1
        elif guess_int < random_num:
            print("Your guess is too low, try again!")
            guess_count += 1


    else:
        print("This is not a valid input!")


if guess_count == 10:
    print("\nWait... you ran out of guesses! That's it, you lose!")

File: java/Game.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

import java.util.Random;

public class Game {

    public static void main(String[] args) throws IOException {
        System.out.println("Welcome to the number guessing game imlemented in Python.");
        System.out.println("Please guess a number that has been randomly chosen between 1 and 100.");
        System.out.println("You only have 9 guesses, so make them count!");
        System.out.println("Type 'quit' as your guess at any time to quit the game.");

        Random random = new Random();
        int randomNum = random.nextInt(99) + 1;

        BufferedReader input = new BufferedReader(new InputStreamReader(System.in));

        int guessCount = 1;

        while (guessCount < 10) {
            System.out.print("\nGuess number " + guessCount + ": ");
            String guessStr = input.readLine();

            if (guessStr.equals("quit")) {
                System.out.println("Bye!");
                guessCount = 11;
            } else {
                try {
                    int guessInt = Integer.parseInt(guessStr);

                    if (guessInt < 1 || 100 < guessInt) {
                        System.out.println("Please guess a number between 1 and 100");
                    } else if (guessInt == randomNum) {
                        System.out.println("Congradulations! You guesses the correct number!");
                        guessCount = 11;
                    } else if (guessInt > randomNum) {
                        System.out.println("Your guess is too high, try again!");
                        guessCount += 1;
                    } else if (guessInt < randomNum) {
                        System.out.println("Your guess is too low, try again!");
                        guessCount += 1;
                    }
                } catch (NumberFormatException e) {
                    System.out.println("This is not a valid input!");
                }
            }
        }

        if (guessCount == 10) {
            System.out.println("\nWait... you ran out of guesses! That's it, you lose!");
        }
    }

}